sdk/include/ndk/pstypes.h

```diff
@@ -906,7 +906,7 @@ typedef struct _PROCESS_PRIORITY_CLASS
 {
     BOOLEAN Foreground;
     UCHAR PriorityClass;
-} PROCESS_PRIORITY_CLASS, *PPROCESS_PRIORITY_CLASS;
+}__attribute__((aligned(sizeof(ULONG)))) PROCESS_PRIORITY_CLASS, *PPROCESS_PRIORITY_CLASS;
 
```

Comment:
Is __attribute__ available in MSVC?

Reply:
I don't know, this structure is not available publicly as it's part of NDK. But with the investigations being done and the testcase pushed to master, it seems like Windows uses the very same attribute.

Reply:
It's not. You want `DECLSPEC_ALIGN` instead for the cross-platform version.
Also, check out this comment for correct placement of the attribute: https://git.reactos.org/?p=reactos.git;a=blob;f=sdk/include/xdk/ntbasedef.h;hb=ae0e0569d5015f31b79a599a903bb9fb43b74e95#l241